test/parallel/test-cluster-rr-distribute.js

```diff
@@ -0,0 +1,69 @@
+// Flags: --expose_internals
```

Comment:
nit... while `--expose_internals` works, `--expose-internals` is more common/idiomatic

Reply:
Done.